components/personCard/personCard.module.scss

```diff
@@ -4,14 +4,14 @@
   width: 550px;
   height: auto;
   max-height: 220px;
-  min-height: 220px;
```

Comment:
what's the reason for removing it?

Reply:
Did it by mistake. Although I cant see any change to the card. Is it important to set min and max to the same size?
And if it does I'll make another PR to change it back.

Reply:
I'm not sure - @MichalPorag any comment on that?

also, you can make changes to the existing PR. As you long as you commit them to the branch that you used for the PR (in this case, you `master` branch), they'll be automatically added to the PR

Reply:
Awesome thx for the tip I'll wait for Michal's answer.